addons/l10n_in_edi_ewaybill/models/res_config_settings.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, _
+from odoo.exceptions import UserError
+from odoo.tools import html_escape
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    l10n_in_edi_ewaybill_username = fields.Char("Indian EDI Stock username",
+        related="company_id.l10n_in_edi_ewaybill_username", readonly=False)
```

Comment:
(no need for compute_sudo something then?)

Reply:
why compute_sudo is needed?
because we use group "base.group_system"!

Reply:
no, for account user we use this with sudo